pages/integrations/phabricator.md.erb

```diff
@@ -0,0 +1,85 @@
+# Phabricator
+
+[Phabricator][] can trigger Buildkite on new revisions through Habormaster.
+
+<%= toc %>
+
+## Prerequisites
+
+Check that your repository is activated in your Phabricator instance and
+Habormaster is installed. Configure a pipeline for that repository in Buildkite.
+You also need admin accces in your Phabricator and your Buildkite organization.
+
+## Step 1: New Phabricator Build Step
+
+Create a new Build Plan in Harbormaster. Inside Habormaster open "Manage
+Build Plans". Press "Create Build Plan" located in the upper right.
+
+<%= image "phabricator_create_build_plan.png", alt: "Creating a new Build Plan in Phabricator" %>
+
+Provide a name then press "Create Build Plan".
+
+Now, add a step to the Build Plan. A "Build Step" calls one BuildKite
+"pipeline". If you need to call multiple pipelines then create multiple Build
+Steps.
+
+<%= image "phabricator_run_in_buildkite.png", alt: "Selecting Builkite for this Build Step" %>
+
+Press "Add Build Step" on the next screen. Then select "Build with Buildkite".
+
+<%= image "phabricator_blank_build_step.png", alt: "Incomplete Build Step" %>
+
+Keep this screen open while you configure Buildkite.
+
+## Step 2: Configure Buildkite Notification Webhook
+
+Phabricator and Buildkite integrate via webhooks. Phabricator triggers
+Buildkite via webhook, then Buildkite reports the status via webhook. You need
+to create an API Access Token in Buildkite to configure Phabricator.
```

Comment:
What do you think about moving this paragraph up into the introduction above the TOC? It would be good to have a bit more up there and this gives some info about how the things all work? 🤔